src/clr-angular/utils/animations/expandable/expandable.ts

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (c) 2016-2019 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import { Component, ElementRef, HostBinding, HostListener, Input } from '@angular/core';
+
+import { animate, style, transition, trigger } from '@angular/animations';
+import { DomAdapter } from '../../dom-adapter/dom-adapter';
+
+@Component({
+  selector: 'clr-expandable',
+  template: `
+    <ng-content></ng-content>
+  `,
+  styles: [
+    `
+    :host {
+      display: block;
+      overflow: hidden;
+    }
+  `,
+  ],
+  host: {
+    class: 'expandable',
```

Comment:
We're not using the `clr-` prefix everywhere (see datagrid) though we are using it some places (see wizard). My point was that we advertise using class-based selectors where needed, not element selectors. Also as this is a building-block type of component, it could be used in other components or even standalone. Then it may get some styling.
I am not huge fan of doing stuff "for the unclear future cases", so I'm okay to remove it.

Reply:
The prefixing is still a work in progress. https://github.com/vmware/clarity/issues/2162 The main issue with not prefixing is a generic selector name like `expandable` has a high probability of it getting styles globally that we didnt intend to have. Either way once we turn on shadow DOM encapsulation we will only need to prefix public selectors.

Reply:
Removed.